sonar-project.properties

```diff
@@ -0,0 +1,13 @@
+sonar.projectKey=keanu:keanu-project
+sonar.projectName=Keanu
+sonar.projectVersion=0.0.5-SNAPSHOT
```

Comment:
If we need to update this then we should add it to the release process.

Reply:
I think it actually overrides this value with what it finds in the build.gradle. I'll remove it and make sure.

Reply:
Yep, it ran fine without the `sonar.projectVersion`